src/node.cc

```diff
@@ -4993,8 +4993,40 @@ int Start(int argc, char** argv) {
 namespace lib {
 
 struct CmdArgs {
+  /**
+   * @brief CmdArgs builds a continuous adjacent char buffer (argv) from a vector of strings
+   * and stores it in the static cmd_args variable
```

Comment:
Technically, it's not really a "static" variable, is it? Looks like a simple member variable to me.

Reply:
right, I fixed the documentation